docs/features/Authorization.md

```diff
@@ -67,6 +69,15 @@ There are also some special scopes
 **Cronjobs/Actions:RUN** The ability to manually run an action or cronjob
 **Objects :REVERT ** The ability to manually revert an object to an earlier version
 **Plugins :UPDATE  ** The ability to manually update a plugin to a newer version
+**admin.method ** The ability to access all /api endpoints based on request method.
+
+The default admin user has all admin.method scopes (admin.GET, admin.POST etc) by default.
+
+### API scopes
+
+API scopes should  be assigned to users through their SecurityGroups which can be added through the admin user interface. If you would like to work with an /api endpoint for a specific Schema the scope would be schemas.https://example.com/schema/example.schema.json.GET
+
+If you have an endpoint with a configured proxy you need the scope of the source to work with it, for example scope: sources.https://example.com/source/api.petstore.source.json.GET
```

Comment:
Same as schemas, sources.referenceOfSource.GET to clarify that this is a reference of the Source